src/ToDoList.tsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
+import {useState} from "react";
+import Modal from '@mui/material/Modal';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
 import './ToDoList.css';
 
 export const ToDoList = () => {
+    const [openWindow, setOpenWindow] = useState(false);
```

Comment:
bool variables' names should start with `is`, `are`, `has`, `have` prefixes
`openWindow ` seems like function name